### PR TITLE
Fixes of autotests C353639, C353640, C358140

### PR DESCRIPTION
**Pull request by Sherzod-Kenjaev**
https://foliotest.testrail.io/index.php?/cases/view/353639
https://foliotest.testrail.io/index.php?/cases/view/353640
https://foliotest.testrail.io/index.php?/cases/view/358140

### CODE REVIEW

File: cypress/e2e/inventory/search/browse-call-number-with-space.cy.js [hunk 1]
```diff
@@ -1,54 +1,73 @@
-import getRandomPostfix from '../../../support/utils/stringTools';
-import DevTeams from '../../../support/dictionary/devTeams';
-import TestTypes from '../../../support/dictionary/testTypes';
-import InventoryActions from '../../../support/fragments/inventory/inventoryActions';
-import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
-import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
-import BrowseCallNumber from '../../../support/fragments/inventory/search/browseCallNumber';
-import TopMenu from '../../../support/fragments/topMenu';
-import BrowseContributors from '../../../support/fragments/inventory/search/browseContributors';
+import getRandomPostfix from "../../../support/utils/stringTools";
```
Comment (TetianaParanich) [inline cypress/e2e/inventory/search/browse-call-number-with-space.cy.js:1]: I think it's better to use single quotes)

Comment (Sherzod-Kenjaev) [reply cypress/e2e/inventory/search/browse-call-number-with-space.cy.js:1]: Yeah you are right) I forgot to change the settings of "Prettier" in vs code, it changed quotes from single to double automatically and other parts as well), I'll fix it thanks.

**Review by penfriend — CHANGES_REQUESTED** [review]

File: cypress/support/fragments/inventory/search/browseContributors.js [hunk 2]
```diff
@@ -40,44 +46,46 @@ export default {
...
+      recordSearch.fillIn(' '),
     ]);
     cy.expect([
-      Form().find(inventorySearch).exists(),
-      Form().find(browseButton).exists(),
-      Form().find(Button('Reset all')).exists(),
-      Form().find(Button({ id: 'accordion-toggle-button-nameType' })).exists(),
+      Form().find(recordSearch).exists(),
+      Form().find(searchButtonDisabled).exists(),
+      Button({ id: "accordion-toggle-button-nameType" }).exists(),
```
Comment (TetianaParanich) [inline cypress/support/fragments/inventory/search/browseContributors.js:77]: please, change double to single quotes

File: cypress/e2e/inventory/search/browse-call-number-with-space.cy.js [hunk 3]
```diff
@@ -34,21 +39,29 @@ describe('ui-inventory: search', () => {
...
-
-    cy.loginAsAdmin({ path: TopMenu.inventoryPath, waiter: InventorySearchAndFilter.waitLoading });
   });
 
   afterEach('Deleting user and instance', () => {
     InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(item.itemBarcode);
+    Users.deleteViaApi(testData.user.userId);
   });
 
   it('C358140 Verify that browsing for "call number" with "space" value will get the correct result (spitfire)', { tags: [DevTeams.spitfire, TestTypes.smoke] }, () => {
```
Comment (TetianaParanich) [inline cypress/e2e/inventory/search/browse-call-number-with-space.cy.js:57]: test is failed, please check it

File: cypress/e2e/inventory/search/browse-contributors-exact.cy.js [hunk 4]
```diff
@@ -51,6 +51,7 @@ describe('ui-inventory: search', () => {
   });
 
   it('C353639 Browse contributors with exact match query (spitfire)', { tags: [testType.smoke, devTeams.spitfire] }, () => {
```
Comment (TetianaParanich) [inline cypress/e2e/inventory/search/browse-contributors-exact.cy.js:53]: test is failed, please check it

File: cypress/e2e/inventory/search/browse-contributors-non-exact.cy.js [hunk 5]
```diff
@@ -51,6 +51,7 @@ describe('ui-inventory: search', () => {
   });
 
   it('C353640 Browse contributors with non exact match query (spitfire)', { tags: [testType.smoke, devTeams.spitfire] }, () => {
```
Comment (TetianaParanich) [inline cypress/e2e/inventory/search/browse-contributors-non-exact.cy.js:53]: test is failed too, please check it

File: cypress/support/fragments/inventory/search/browseContributors.js [hunk 6]
```diff
@@ -40,44 +46,46 @@ export default {
...
+      PaneContent('Choose a filter or enter a search query to show results.').exists(),
     ]);
   },
 
   browse(contributorName) {
-    cy.do([
-      inventorySearch.fillIn(contributorName),
-      browseButton.click(),
-    ]);
+    cy.do([recordSearch.fillIn(contributorName), searchButton.click()]);
```
Comment (TetianaParanich) [inline cypress/support/fragments/inventory/search/browseContributors.js:88]: Do you need array in this place?

File: cypress/support/fragments/inventory/search/browseContributors.js [hunk 7]
```diff
@@ -114,9 +122,7 @@ export default {
   },
 
   openInstance(contributor) {
-    cy.do([
-      MultiColumnListCell(contributor.name).click(),
-    ]);
+    cy.do([MultiColumnListCell(contributor.name).click()]);
```
Comment (TetianaParanich) [inline cypress/support/fragments/inventory/search/browseContributors.js:125]: Do you need array in this place?

File: cypress/e2e/inventory/search/browse-call-number-with-space.cy.js (hunk 3 above)
Comment (TetianaParanich) [reply cypress/e2e/inventory/search/browse-call-number-with-space.cy.js:57]: The test is passed https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/526/allure/

File: cypress/e2e/inventory/search/browse-contributors-exact.cy.js (hunk 4 above)
Comment (TetianaParanich) [reply cypress/e2e/inventory/search/browse-contributors-exact.cy.js:53]: The test is passed https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/527/allure/

**Review by TetianaParanich — APPROVED** [review]

**Pull request closed without merging**